internal/web/commands.go

```diff
@@ -90,32 +88,32 @@ func buildISO(containerImage, outputDir, artifactName string, ws io.Writer) erro
 		ContainerImage: fmt.Sprintf("docker://%s", containerImage),
 	}
 
-	// Create the config
-	config := schema.Config{
-		State: outputDir,
-		ISO: schema.ISO{
-			OverrideName: artifactName,
-		},
+	// Read the config using the shared config package
+	config, _, err := config.ReadConfig("", "", nil)
```

Comment:
Initially I thought the reason it as failing was because I failed to set the cloud config. I tried to build the config using `ReadConfig` and this is the reason I extracted it to a separate package in this PR. I still think it's better to re-use this function but I can revert this change if if feels irrelevant.